### DISCUSSION TITLE
How can I color subindex in Tex?

### DISCUSSION THREAD

**Discussion by billy-ijk** [3b1b/manim#1868] [Q&A] [answered]
Hi, I'm trying to change index color in `text = Tex("$\\dim_{\\mathbb{R}} (\\mathbb{C})$)`, specifically that R. Can anyone help me?

**Comment by utof** [comment, marked as answer]
is this what you're looking for?
```
class sorry_for_delay(Scene):
    def construct(self):
        text = Tex("$\\dim_{\\mathbb{R}} \\mathbb{C}$")
        text[0][3].set_color(YELLOW) 
        self.play(FadeIn(text))
```
![image](https://user-images.githubusercontent.com/38869938/196307267-47b35942-873b-4d72-abbc-e58908503818.png)
